[Test] 이슈 발행 테스트코드 작성

asd42270 opened the pull request:
# Pull Request 요약

- 이슈 발행 테스트 코드를 작성합니다.

## 변경 사항

- 테스트 코드 및 아스키닥 문서 작성

## 관련 이슈

- #103 

## 개발 유형

- [ ] 버그 수정
- [ ] 새로운 기능 개발
- [ ] 리팩토링
- [x] 테스트 코드 작성
- [x] 문서 업데이트
- 기타 (아래에 설명 추가)

## 작업 기간

- 작업 시작일: (YYYY-MM-DD)
- 작업 종료일: (YYYY-MM-DD)

## 유의사항

- 이 PR로부터 발생할 수 있는 유의사항이 있나요? 이 PR이 다른 부분에 미칠 수 있는 영향 등을 설명하세요.

## 참고문헌

- 이 PR을 작성하며 참조한 문헌이나 가이드가 있다면 여기에 링크를 추가하세요.

asd42270 committed:
[feat] 유지관리 이슈 테스트 코드를 작성한다.
- ManagerArticleIssuePublishServiceTest:
성공 테스트: 메서드 실행 여부를 파악한다.
실패 테스트: 게시글 조회 시 존재하지 않는 게시글은 오류가 출력된다.

asd42270 committed:
Merge remote-tracking branch 'origin/feature/92' into feature/92

asd42270 committed:
파일 위치를 옮겨요

asd42270 committed:
[feat] 이슈 발행 테스트 코드를 작성한다.
- articleissue-api : 이슈 발핼 api 문서를 작성합니다.
- ArticleIssueControllerDocsTest: 아스키닥 문서 작성 컨트롤러 테스트 코드를 작성했습니다.
- ManagerArticleIssuePublishServiceTest : 유지 관리 이슈 발행 테스트 코드를 작성합니다.
- RepairArticleIssuePublishServiceTest : 유지 보수 이슈 발행 테스트 코드를 작성합니다.

Galmaeki commented on src/test/java/com/final_10aeat/domain/issue/docs/ArticleIssueControllerDocsTest.java:
```diff
@@ -0,0 +1,111 @@
+        //given
+        ArticleIssuePublishRequestDto issueRequest = new ArticleIssuePublishRequestDto(
+                "title", "content");
+        // when
```
이부분은 given 과정에 해당한다고 생각합니다!
아래 주석을 // when & then으로 바꾸는건 어떻게 생각하시나요?

Galmaeki commented on src/test/java/com/final_10aeat/domain/issue/unit/RepairArticleIssuePublishServiceTest.java:
```diff
@@ -0,0 +1,98 @@
+    private final String email = "test@naver.com";
+    private final String password = "spring";
+
+    ArticleIssuePublishRequestDto requestDto = new ArticleIssuePublishRequestDto(issueTitle, issueContent);
```
접근제어자를 주지 않은 이유가 있으실까요?

Galmaeki reviewed (changes requested):
고생 많으셨습니다!
코멘트 확인 부탁드림다!

ghrltjdtprbs reviewed (approved):
수고하셨습니다! 👍👍

asd42270 committed:
[feat] pr 수정사항 반영
- ArticleIssueControllerDocsTest: when&then으로 주석 변경
- ManagerArticleIssuePublishServiceTest :접근 제어자 설정, when&then으로 주석 변경
- RepairArticleIssuePublishServiceTest : 접근 제어자 설정, when&then으로 주석 변경

Galmaeki commented on src/test/java/com/final_10aeat/domain/issue/docs/ArticleIssueControllerDocsTest.java:
```diff
@@ -0,0 +1,109 @@
+        // when&then
+        doNothing().when(articleIssueService).manageIssuePublish(eq(issueRequest), any(Long.class), any(Manager.class));
+
+        mockMvc.perform(RestDocumentationRequestBuilders.post("/managers/articles/manage/issue/{manage_article_id}", 1)
```
59번 라인에 When & Then을 달아야 함다ㅠ....

Galmaeki reviewed (changes requested):
보완 코멘트 남겼슴다!
확인 부탁드려오!

asd42270 committed:
[fix] pr 수정사항 반영
- ArticleIssueControllerDocsTest: when&then으로 주석 위치 변경

Galmaeki reviewed (approved):
고생하셨습니다
